src/actions/ScanActions.tsx

```diff
@@ -167,40 +166,12 @@ export const addressWarnings = async (parsedUri: any, currencyCode: string) => {
   return approve
 }
 
-export function parseScannedUri(data: string, customErrorTitle?: string, customErrorDescription?: string): ThunkAction<Promise<unknown>> {
+export function handleWalletUris(coreWallet: EdgeCurrencyWallet, parsedUri: EdgeParsedUri, fioAddress?: string): ThunkAction<Promise<unknown>> {
```

Comment:
In general, please call `EdgeCurrencyWallet` instances `wallet`, not `coreWallet`. Aside from legacy places we need to clean up, a "wallet" is an Edge wallet in the Edge app. A second word is useful if it adds meaning, like "sourceWallet" or "bestWallet". I can also be useful if there is another type of non-Edge wallet, like a "wcApiWallet" or such, but "coreWallet" just adds visual noise, since we use this type everywhere possible.

Reply:
I vividly recall a discussion about this and agreeing to `coreWallet` as accepted nomenclature after considering `edgeWallet` and `wallet`. Especially while we still have objects called `wallet` that are actually `GuiWallet`. Code filled with single words is just harder to grep through. It's one of the reasons edge-core-js is so hard to read through with object params like `state`, `ai`, `plugins` that exist everywhere.

How about this as a compromise:

Local variables which are within a single file can be short single words (ie `wallet`) and I'll make the change you requested above. It's less relevant to grep through code searching for these terms. However for object params that are shared between files, can we mandate multiple word names to provide uniqueness? This would REALLY help read through projects like the core much better.